tests/partial_props.rs

```diff
@@ -0,0 +1,54 @@
+use confique::Config;
+use pretty_assertions::assert_eq;
+
+#[test]
+fn partial_props() {
+    #[allow(dead_code)]
+    #[derive(Config)]
+    #[config(partial_attr(derive(derive_more::Debug)))]
+    struct Foo {
+        #[config(default = 1, partial_attr(debug("test {bar:?}")))]
+        bar: u32,
+    }
+
+    use confique_partial_foo::PartialFoo;
+    let partial_foo = PartialFoo { bar: Some(1) };
+    assert_eq!(
+        "PartialFoo { bar: test Some(1) }",
+        format!("{partial_foo:?}")
+    );
```

Comment:
Usually you pass the "actual" value first, with the "expected" one second, no? Also trailing comma

```suggestion
    assert_eq!(
        format!("{partial_foo:?}"),
        "PartialFoo { bar: test Some(1) }",
    );
```

Reply:
Probably, I'm just used to JUnit where it enforces expected-actual, unfortunately. Fixed.